src/3_PB/PianoDiProgetto/sections/ConsuntivoSprint/QuindicesimoSprint.typ

```diff
@@ -0,0 +1,47 @@
+#import "../../const.typ": Re_cost, Am_cost, An_cost, Ve_cost, Pr_cost, Pt_cost
+#import "../../functions.typ": rendicontazioneOreAPosteriori, rendicontazioneCostiAPosteriori, glossary, team
+
+
+== Quindicesimo #glossary[sprint]
+
+*Inizio*: Venerdì 29/03/2024
+
+*Fine*: Giovedì 04/04/2024 
+
+#rendicontazioneOreAPosteriori(sprintNumber: "15")
+
+#rendicontazioneCostiAPosteriori(sprintNumber: "15")
+
+=== Analisi a posteriori
+
+La retrospettiva dell'ultimo #glossary[sprint] prima della seconda revisione #glossary[PB] mostra che ancora una volta il team è stato in grado di rispettare le ore preventivate, impiegando 29.5 ore produttive per portare a termine le attività previste; non sorprende che la maggior parte delle ore siano state impiegate dai Verificatori, dato che tra gli obiettivi della pianificazione vi erano la conclusione dello sviluppo del prodotto software e la revisione della #glossary[documentazione] in stile #glossary[walkthrough]. A tal proposito il team è riuscito a completare lo sviluppo del prodotto nei tempi previsti e lo ha presentato alla Proponente in data 02/04/2024 per effettuare i test di accettazione ed ottenere la validazione dello stesso come #glossary[MVP]. La retrospettiva evidenzia anche che, al termine dei 7 #glossary[sprint] effettuati nel periodo di tempo trascorso tra l'#glossary[RTB] e la #glossary[PB] (il che corrisponde ad uno #glossary[sprint] in più rispetto a quelli che il team aveva previsto di dover utilizzare), le ore produttive rimanenti ammontano a 15 su 570 totali (2.6%) e, di conseguenza, il budget rimanente è di 282.5€ su 11.070€ totali (2.6%) a disposizione. 
```

Comment:
```suggestion
La retrospettiva dell'ultimo #glossary[sprint] prima della seconda revisione #glossary[PB] mostra che ancora una volta il team è stato in grado di rispettare le ore preventivate, impiegando 29.5 ore produttive per portare a termine le attività previste; non sorprende che la maggior parte delle ore siano state impiegate dai Verificatori, dato che tra gli obiettivi della pianificazione vi erano la conclusione dello sviluppo del prodotto software e la revisione della #glossary[documentazione] in stile #glossary[walkthrough]. A tal proposito il team è riuscito a completare lo sviluppo del prodotto nei tempi previsti e lo ha presentato alla Proponente in data 02/04/2024 per effettuare i test di accettazione ed ottenere la validazione dello stesso come #glossary[MVP]. La retrospettiva evidenzia anche che, al termine dei 7 #glossary[sprint] effettuati nel periodo di tempo trascorso tra l'#glossary[RTB] e la #glossary[PB] (corrispondente ad uno #glossary[sprint] in più rispetto a quanto il team aveva previsto di dover utilizzare), le ore produttive rimanenti ammontano a 15 su 570 totali (2.6%) e, di conseguenza, il budget rimanente è di 282.5€ su 11.070€ totali (2.6%) a disposizione. 
```

Reply:
ripensandoci, ho tolto il "(il che corrisponde ad uno #glossary[sprint] in più rispetto a quelli che il team aveva previsto di dover utilizzare)" perchè mi sono resa conto che non è spiegato/motivato da nessuna parte nel PdP; cioè dovrei andare in uno dei consuntivi precedenti e dire "il team si è reso conto di necessitare di uno sprint aggiuntivo per portare a termine gli obiettivi del progetto e ha deciso di.. per questi motivi.."; posso anche farlo in realtà, ma se anche lasciassimo così non se ne accorgerebbe nemmeno.

Reply:
Si sono daccordo per l'averlo tolto.